ext/rbkit_event_packer.c

```diff
@@ -102,13 +102,17 @@ static void pack_hash_event(rbkit_hash_event *event, msgpack_packer *packer) {
 
 static void pack_object_space_dump_event(rbkit_object_space_dump_event *event, msgpack_packer *packer) {
   rbkit_object_dump *dump = event->dump;
-  msgpack_pack_map(packer, 4);
+  msgpack_pack_map(packer, 5);
   pack_event_header(packer, event->event_header.event_type);
 
   // Incrementing integer holding the correlation_id
   // indicating the event which the message belongs to
   msgpack_pack_int(packer, rbkit_message_field_correlation_id);
   msgpack_pack_int(packer, event->correlation_id);
+  
+  // dump total number of messages in batch
+  msgpack_pack_int(packer, rbkit_message_field_complete_message_count);
+  msgpack_pack_int(packer, event->object_count);
```

Comment:
@gnufied Should we call this `object_count` instead of `complete_message_count` ? Because this number does not tell us how many messages are going to come, instead it tells us only the number of object infos we have serialized.